Userland/Applications/Browser/Tab.cpp

```diff
@@ -408,12 +408,17 @@ Optional<URL> Tab::url_from_location_bar(MayAppendTLD may_append_tld)
 
     StringBuilder builder;
     builder.append(text);
+
     if (may_append_tld == MayAppendTLD::Yes) {
-        // FIXME: Expand the list of top level domains.
```

Comment:
Maybe we can fetch this list from [IANA](https://data.iana.org/TLD/tlds-alpha-by-domain.txt)?

Reply:
Which list?